src/dsp_tools/commands/xmlupload/write_diagnostic_info.py

```diff
@@ -2,22 +2,24 @@
 
 import json
 from datetime import datetime
+from pathlib import Path
 
 from loguru import logger
 
 from dsp_tools.commands.xmlupload.upload_config import DiagnosticsConfig
 
 
-def write_id2iri_mapping(
-    id2iri_mapping: dict[str, str],
-    shortcode: str,
-    diagnostics: DiagnosticsConfig,
-) -> None:
+def write_id2iri_mapping(id2iri_mapping: dict[str, str], shortcode: str, diagnostics: DiagnosticsConfig) -> None:
     """Writes the mapping of internal IDs to IRIs to a file."""
     timestamp = datetime.now().strftime("%Y-%m-%d_%H%M%S")
     servername = diagnostics.server_as_foldername
     id2iri_filename = f"id2iri_{shortcode}_{servername}_{timestamp}.json"
```

Comment:
This name is fine for the cwd, but for the .dsp-tools directory we should put the timestamp in front, otherwise they won't be sorteable by timestamp.